### DISCUSSION TITLE
Add disable "long press new tab" in settings > behavior

### DISCUSSION THREAD

**Discussion by gety9** [plateaukao/einkbro#281] [Ideas]
New tab (+ icon) is one of the most frequently pressed icons in tool bar and currently it has double function - regular press for new tab, long press for new window / app instance.

I never use second but many times it was accidently triggered, it will be great if we can add setting to disable long press functionality.

**Comment by plateaukao** [comment]
For adding an option to disable long click feature for + button, since it's not a common request or behavior to change, I may not add it. You could consider to configure long press time on your device, so that you won't mis-touch it too often.

--
To adjust the touch and hold delay:
Open your device's Settings app . (need to launch the hidden System Setting app. You could install [my another app](https://github.com/plateaukao/OpenSettings) to do so)
Tap Accessibility. Touch & hold delay.
Select Short, Medium, or Long.

**Comment by gety9** [comment]
Agree, closing.
